stats/bin/migrate.js

```diff
@@ -1,12 +1,8 @@
 import {
-  DATABASE_URL,
-  EVALUATE_DB_URL
+  DATABASE_URL
 } from '../lib/config.js'
-import { migrateWithPgConfig as migrateEvaluateDB } from 'spark-evaluate/lib/migrate.js'
 import { migrateWithPgConfig as migrateStatsDB } from '@filecoin-station/spark-stats-db-migrations'
 
-console.log('Migrating spark_evaluate database', EVALUATE_DB_URL)
-await migrateEvaluateDB({ connectionString: EVALUATE_DB_URL })
```

Comment:
In prod, this shouldn't try to migrate the evaluation db, which is connected to read only

Reply:
since we removed `npm run migrate` from the stats production setup, I added evaluation db migration back here